website/docs/jellyfish/api/loan.md

```diff
@@ -269,12 +269,55 @@ Creates a vault transaction.
 interface loan {
   createVault (vault: CreateVault, utxos: UTXO[] = []): Promise<string>
 }
+
 interface CreateVault {
   ownerAddress: string
   loanSchemeId?: string
 }
+
+interface UTXO {
+  txid: string
+  vout: number
+}
+```
+
+## depositToVault
+
+Deposit to vault.
+
+```ts title="client.loan.depositToVault()"
+interface loan {
+  depositToVault (vault: DepositVault, utxos: UTXO[] = []): Promise<string>
+}
+
+interface DepositVault {
+  id: string
```

Comment:
You probably need to change this to vaultId like
https://github.com/DeFiCh/jellyfish/pull/661/files#diff-3e4e57ab619608405a6f96e0ac91e233a158743d1f3ed54e16cdadffec038e08R315